components/movementsensor/imuvectornav/imu.go

```diff
@@ -122,20 +114,23 @@ func newVectorNav(
 		return nil, err
 	}
 
-	speed := *newConf.Speed
-	if speed == 0 {
-		speed = 8000000
+	baudRate := *newConf.BaudRate
+	if baudRate == 0 {
+		baudRate = 9600
```

Comment:
add an info level message here to declare that we are setting the default baudRate to 9600. Also let's make a global constant variable `DefaultBaudRate` and set it to 9600 and use it here